TerminalDocs/customize-settings/actions.md

```diff
@@ -698,7 +695,7 @@ _This command is not currently bound in the default settings_.
 > [!IMPORTANT]
 > This feature is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
 
-### Minimize to tray ([Preview](https://aka.ms/terminal-preview))
+### Minimize to tray
 
 This will hide the currently focused window from the Taskbar and instead will be accessible from the system tray icon. This action will only be useable when the tray icon is visible through one of the two global settings `minimizeToTray` or `alwaysShowTrayIcon`.
```

Comment:
@leonmsft to confirm: i think we actually did NOT ship the `minimizeToTray` action at all!

Reply:
yup, didn't include the minimize to tray action at all, was supposed to remove that with this docs pr https://github.com/MicrosoftDocs/terminal/pull/387